.gitignore

```diff
@@ -15,6 +15,7 @@ dist/
 .netlify
 matico_charts/package-lock.json
 .vscode/settings.json
+.idea/
```

Comment:
I added `.idea/` to the `.gitignore` file so that any user with a JetBrains IDE isn't being asked to include configuration files in their commits.